developer/architecture/routing.md

```diff
@@ -64,7 +64,7 @@ _Default permissions are not added automatically, as there will be a UI componen
 
 `Accounts.onCreateUser` adds default roles need for products, profiles, and checkout routes automatically if none are defined for the Shop.
 
-```javascript
```

Comment:
Yeah, both will work fine.  I just wanted to standardize and went with the short form for quicker typing.  Same deal with `bash` and `sh`.